docs/source/usage.rst

```diff
@@ -0,0 +1,185 @@
+
+__ https://github.com/peaceiris/actions-gh-pages
+
+::
+
+    name: publish-pydoctor-apidocs
+    on:
+    - push
+
+    jobs:
+      deploy:
+        runs-on: ubuntu-latest
+
+        steps:
+        - uses: actions/checkout@master
+        - name: Set up Python 3.8
+          uses: actions/setup-python@v2
+          with:
+            python-version: 3.8
+        
+        - name: Install package
+          run: |
+            python -m pip install --upgrade pip setuptools wheel
+            python -m pip install .
+            python -m pip install pydoctor
+
+        - name: Generate pydoctor documentation
+          run: |
+            # Allow pydoctor to exit with non-zero status code
+            set +e
+
+            # Run pydoctor build
+            pydoctor \
+                --add-package=(packagedirectory) \
+                --project-name=(projectname) \
+                --project-url=https://github.com/$GITHUB_REPOSITORY \
+                --html-viewsource-base https://github.com/$GITHUB_REPOSITORY/tree/$GITHUB_SHA \
+                --make-html \
+                --html-output=./apidocs \
+                --project-base-dir "$(pwd)" \
+                --docformat=restructuredtext 
+
+        - name: Publish pydoctor documentation to the gh-pages branch
+          uses: peaceiris/actions-gh-pages@v3
+          with:
+            github_token: ${{ secrets.GITHUB_TOKEN }}
+            publish_dir: ./apidocs
+            commit_message: "Generate pydoctor documentation"
+
+.. note:: As mentionned in the ``actions-gh-pages`` documentation, the first workflow run won't actually publish the documentation to Github pages. 
+    Github pages needs to be enabled afterwards in the repo settings, select ``gh-pages`` branch, then re-run your workflow. 
+
+    The website we'll be at https://(user).github.io/(repo)/
+
+Publish your documentation with Sphinx and Read The Docs
+--------------------------------------------------------
+
+.. note:: Documentation to come!
+
+Document part of your package
+-----------------------------
+
+Sometimes, only a couple classes or modules are part of your public API, not all classes and modules need to be documented.  
+
+You can choose to document only a couple classes or modules passed with argument ``--html-subject``.
+
+.. warning:: The ``index.html`` won't be generated, you need to link to the specific HTML page. 
+
+Sphinx Integration
+------------------
+
+Sphinx object inventories can be used to create deep links between API
+documentation generated by pydoctor and by Sphinx.
+
+Linking to external API docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It can link to external API documentation using a Sphinx objects inventory
+with the following cumulative configuration option::
+
+    --intersphinx=http://sphinx-doc.org/objects.inv
```

Comment:
It would help to have an example of pydoctor to sphinx usage in a similar way that we have the sphinx to pydoctor.

Maybe use a real life example using a link to stdlib.

Reply:
I've put https://docs.python.org/3/objects.inv instead